core/src/main/java/org/bitcoinj/crypto/HDPath.java

```diff
@@ -0,0 +1,120 @@
+package org.bitcoinj.crypto;
+
+import java.util.AbstractList;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Representation of HD Key derivation path
```

Comment:
You might want to use some paragraph markup in your JavaDoc.

Reply:
Fixed in force push: https://github.com/bitcoinj/bitcoinj/compare/7e832e4fd77a5cabd1df09caf3da946c4a140167..df2d543d4204193be42424bbaa6c15687cd82910